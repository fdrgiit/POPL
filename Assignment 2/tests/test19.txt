s a b c #
A B C #
s -> a A #
a -> b B #
b -> c C #
c -> #
##

