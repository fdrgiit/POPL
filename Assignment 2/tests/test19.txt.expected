FIRST(s) = { C }
FIRST(a) = { C }
FIRST(b) = { C }
FIRST(c) = { # }
FOLLOW(s) = { $ }
FOLLOW(a) = { A }
FOLLOW(b) = { B }
FOLLOW(c) = { C }
